pexec/managed_process.go

```diff
@@ -32,6 +32,10 @@ type ManagedProcess interface {
 	// there's any system level issue stopping the process.
 	Stop() error
 
+	// Kill will attempt to kill the process group and not wait for completion. Only use this if
```

Comment:
```suggestion
	// KillGroup will attempt to kill the process group and not wait for completion. Only use this if
```
Surprised lint passes.